components/atlas-uvs/examples/dynamic/index.html

```diff
@@ -3,7 +3,7 @@
     <title>Dynamic</title>
     <meta name="description" content="Dynamic texture atlas."></meta>
     <script src="https://aframe.io/releases/0.8.2/aframe.min.js"></script>
-    <script src="../../dist/aframe-atlas-uvs-component.min.js"></script>
+    <script src="https://unpkg.com/aframe-atlas-uvs-component@2.1.0/dist/aframe-atlas-uvs-component.min.js"></script>
```

Comment:
This is using a very old A-Frame version. Can we update to 1.2.0?

Reply:
yes upgraded